client/src/App.js

```diff
@@ -2,18 +2,24 @@ import React from "react";
 import Navbar from "./components/Navbar/Navbar";
 import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
 import Home from "./pages/Home/Home";
-import AboutUs from "./pages/Home/AboutUs/AboutUs";
+import About from "./pages/About/About";
 import Register from "./pages/RegisterPage/Register";
 import "./App.css";
+import Resources from "./pages/Resources/Resources";
+import Contact from "./pages/Contact/Contact";
+import Login from "./pages/Login/Login";
 
 function App() {
 	return (
 		<>
 			<Router>
 				<Navbar />
 				<Switch>
-					<Route path="/" exact component={Home} />
-					<Route path="/about" component={AboutUs} />
+					<Route path="/home" exact component={Home} />
```

Comment:
Hi Andrejs
please make it 
```js
<Route path="/" exact component={Home} />
```
so that home page loaded once user opens up the web site